forest/src/cli/chain_cmd.rs

```diff
@@ -82,18 +85,31 @@ impl ChainCommands {
 
                 let epoch = tipset.unwrap_or(chain_head.epoch());
 
+                let now = OffsetDateTime::now_utc();
+
+                let month = now.month() as u8;
+                let month_string = if month < 10 {
+                    format!("0{}", month)
+                } else {
+                    month.to_string()
+                };
+                let year = now.year();
+                let day = now.day();
+                let network_name = cfg.chain.name.clone();
+
+                let mut vars = HashMap::new();
+                vars.insert("year".to_string(), year.to_string());
+                vars.insert("month".to_string(), month_string.clone());
+                vars.insert("day".to_string(), day.to_string());
+                vars.insert("network".to_string(), network_name.clone());
+                vars.insert("heigth".to_string(), epoch.to_string());
+
                 let output_path = match output_path {
-                    Some(path) => path.to_owned(),
-                    None => {
-                        let now = OffsetDateTime::now_utc();
-                        format!(
-                            "forest_snapshot_{}-{}-{}_height_{}.car",
-                            now.year(),
-                            now.month(),
-                            now.day(),
-                            epoch,
-                        )
-                    }
+                    Some(path) => strfmt(&path, &vars).unwrap(),
```

Comment:
Is `unwrap` call safe here? If yes some comment would be nice.

Reply:
Added